std/string.d

```diff
@@ -4806,7 +4814,7 @@ unittest
  *  indent = string to use to _indent following lines of the paragraph
  *  tabsize = column spacing of tabs
  * Returns:
- *  The resulting paragraph.
+ *  resulting paragraph as an allocated string
```

Comment:
The first letter after `:` should be capitalized, otherwise it looks odd on the rendered HTML page.